pages/team/[id].tsx

```diff
@@ -0,0 +1,318 @@
+                                            placeholder="GitHub URL"
+                                        />
+                                    </div>
+                                    <button
+                                        type="button"
+                                        onClick={() => {
+                                            const issues = [...values.github_urls]
+                                            issues.splice(index, 1)
+                                            setFieldValue('github_urls', issues)
+                                        }}
+                                    >
+                                        <XIcon className="w-4 text-[red]" />
+                                    </button>
+                                </div>
+                            )
+                        })}
+                        <button
+                            type="button"
+                            onClick={() => setFieldValue('github_urls', [...values.github_urls, ''])}
+                            className="text-red font-bold mb-6"
+                        >
+                            Add GitHub URL
+                        </button>
+
+                        <div className="flex items-center space-x-2 mb-6">
+                            <label
+                                className="text-[16px] font-semibold opacity-75 flex-shrink-0 m-0"
+                                htmlFor="milestone"
+                            >
+                                Milestone
+                            </label>
+                            <Field type="checkbox" name="milestone" id="milestone" />
+                        </div>
+
+                        <div className="flex items-center mt-4 space-x-2 justify-between">
+                            {handleDelete && (
+                                <Button
+                                    onClick={() => {
+                                        if (!confirmDelete) return setConfirmDelete(true)
+                                        handleDelete()
+                                    }}
+                                    type="button"
+                                >
+                                    {confirmDelete ? 'Click to confirm' : 'Delete'}
+                                </Button>
+                            )}
+                            <Button>{submitText}</Button>
+                        </div>
+                    </Form>
+                )
+            }}
+        </Formik>
+    )
+}
+
+const Team = ({ id, organizationId }) => {
+    const [createModalOpen, setCreateModalOpen] = useState(false)
+    const [team, setTeam] = useState(null)
+    const [addTeamMemberOpen, setAddTeamMemberOpen] = useState(false)
+    const handleNewRoadmap = async (values) => {
+        await createRoadmap({ ...values, teamId: team.id.toString(), organizationId })
```

Comment:
```suggestion
        await createRoadmap({ ...values, teamId: team.id.toString()})
```

Reply:
Couldn't create a new roadmap because `organizationId` was getting sent here and being fetching from the cookie, and both were being passed to prisma.